mysql/README.md

```diff
@@ -374,7 +374,7 @@ Returns CRITICAL if the Agent cannot connect to MySQL to collect metrics, otherw
 ## Further Reading
 Read our [series of blog posts][29] about monitoring MySQL with Datadog.
 
-
+[2]: https://github.com/DataDog/integrations-core/blob/master/mysql/datadog_checks/mysql/data/conf.yaml.example
```

Comment:
Is it expected to start at `2` and jump to `13`?

Reply:
nop, this is why there was this PR: https://github.com/DataDog/integrations-core/pull/2750 I need to re-run the script.. conflicts are 😨